test/helpers.spec.ts

```diff
@@ -683,6 +687,18 @@ describe('helpers', () => {
         });
       });
 
+      describe('toNumber()', () => {
```

Comment:
Can you use the correct function name here?

```suggestion
      describe('rangeToNumber()', () => {
```

Reply:
nice catch :+1: